drf_spectacular/plumbing.py

```diff
@@ -50,6 +51,12 @@
     class Choices:  # type: ignore
         pass
 
+UNION_TYPES: Tuple[Any, ...] = (typing.Union,)
+
+# typing.UnionType was added in Python 3.10 for new PEP 604 pipe union syntax
+if hasattr(types, 'UnionType'):
+    UNION_TYPES += (types.UnionType,)  # type: ignore
```

Comment:
This suppression is needed because mypy 0.910 uses an old version of typeshed that doesn't yet know `types.UnionType`, can be dropped with mypy 0.920.

Reply:
Dropped the type suppressions, apparently CI is now using mypy 0.920.

Reply:
I'm not sure why, in my own project, mypy considers `types.UnionType` valid, but not here. Maybe due to testing on Python 3.9. I'll add back the suppression.

> drf_spectacular/plumbing.py:56: error: Module has no attribute "UnionType"; maybe "FunctionType"?